### ISSUE TITLE
[ACTION] Trello component actions

### ISSUE THREAD

**Issue by dylburger** [PipedreamHQ/pipedream#1402]
We need to develop new, [component actions](https://pipedream.com/docs/components/quickstart/nodejs/actions/) for Trello that map to each of the v1 actions currently in the product (Add New Card, etc.)

**Comment by dannyroosevelt** [comment]
@sergioeliot2039 are you actively working on this? Do you need any support? Just checking if it's still on your radar.

**Comment by sergioeliot2039** [comment]
Yes @dannyroosevelt I have this on my radar. No support needed though.

**Comment by dannyroosevelt** [comment]
Great, thanks Sergio
